specification/gsm/resource-manager/Microsoft.ManagementPartner/2018-02-01/gsm.json

```diff
@@ -0,0 +1,403 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/PartnerIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "partner"
+        ],
+        "responses": {
+          "200": {
+            "description": "Get the details of the `Partner`.",
+            "schema": {
+              "$ref": "#/definitions/PartnerResponse"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "summary": "Update a specific `Partner`.",
+        "description": "Update the management partner for the objectId and tenantId.",
+        "operationId": "Partner_Patch",
+        "x-ms-examples": {
+          "PatchPartnerDetails": {
+            "$ref": "./examples/PatchPartnerDetails.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/PartnerIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "partner"
+        ],
+        "responses": {
+          "200": {
+            "description": "Get the details of the `Partner`.",
+            "schema": {
+              "$ref": "#/definitions/PartnerResponse"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Delete a specific `Partner`.",
+        "description": "Delete the management partner for the objectId and tenantId.",
+        "operationId": "Partner_Delete",
+        "x-ms-examples": {
+          "DeletePartnerDetails": {
+            "$ref": "./examples/DeletePartnerDetails.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/PartnerIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "tags": [
+          "partner"
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted the `Partner`."
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ManagementPartner/partners": {
+      "get": {
+        "summary": "Get a specific `Partner`.",
+        "description": "Get the management partner using the objectId and tenantId..",
+        "operationId": "PartnerNoId_Get",
+        "x-ms-examples": {
+          "GetPartnerDetails": {
+            "$ref": "./examples/GetPartnerDetails.json"
```

Comment:
This example is not valid for this operation - "partnerId" is not a parameter here.